sys/cloudformation/stack.yaml

```diff
@@ -578,6 +578,20 @@ Resources:
     Properties:
       UserName: !Ref iamusergithubactions
 
+  # REDIS
+  rediscluster:
+    Type: AWS::ElastiCache::CacheCluster
+    Properties:
+      AutoMinorVersionUpgrade: True
+      CacheNodeType: cache.t3.micro # TODO: cache.t4g.micro
```

Comment:
proposal to switch instance type

Reply:
Is possible to copy data from old instance to new?

Reply:
I'll do an upgrade of the same instance, and then import it so that it'll match the definition. Data will be there, anyway I'll take a snapshot first to avoid any major data loss.

Reply:
change the instance type completed :heavy_check_mark: